README.md

```diff
@@ -83,24 +87,22 @@ In order to use this custom element extension with your commercetools account, y
 You will need to add the custom element to a content type filling in the hosted code URL and the following JSON parameters:
 
 
-| Name     | Type   | Description |
-| -------- | ------ | ----------- |
-| debug    | boolean | (Optional) If present and set to true the debug panel will activate when editing a content item. |
-| multiSelect  | boolean | If set to true, it will be possible to select multiple products. If set to false, it will be possible to only select a single product |
-| commercetools | object | This contains all the details required to connect to the [commercetools API](https://docs.commercetools.com/http-api). The values for this object will be derived from an API client that you configure in commercetools with the exception of the `defaultCulture`. When generating the API client, be sure to select the `view_products` and `view_project_settings` scopes. |
-| commercetools.defaultCulture | string | Set this to the IETF language tag of the language in commercetools to use by default for search. |
-| commercetools.projectKey | string | This is the commercetools project key. |
-| commercetools.clientId | string | This is the commercetools API client ID. |
-| commercetools.secret | string | This is the commercetools API client secret. |
-| commercetools.oauthUrl | string | This is the base URL to use for authenticating with commercetools. |
-| commercetools.apiUrl | string | This is the base URL to use for commercetools API calls. |
-| commercetools.scope | string | This is the list of scopes that the client has. This should include the `view_products` and `view_project_settings` scopes. |
+| Name                         | Type    | Description                                                                                                                                                                                                                                                                                                                                                                    |
+|------------------------------|---------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| multiSelect                  | boolean | If set to true, it will be possible to select multiple products. If set to false, it will be possible to only select a single product                                                                                                                                                                                                                                          |
```

Comment:
```suggestion
| isMultiSelect                  | boolean | If set to true, it will be possible to select multiple products. If set to false, it will be possible to only select a single product                                                                                                                                                                                                                                          |
```